lib/utils/report.mjs

```diff
@@ -115,6 +125,45 @@ export default function report(problem) {
 	result.warn(warningMessage, warningProperties);
 }
 
+/**
+ * @param {Problem} problem
+ */
+function problemRangeDeprecations(problem) {
```

Comment:
question

What about a name which would convey an action?
e.g. `checkPositions` or `enforceConsistency`

Reply:
I've changed it to `checkProblemRangeDeprecations`.
This function is only temporary.

It will change to something like `problemRange` or `getProblemRange`.
I am trying to maintain a parallel with the future name.